openedx/features/course_experience/templates/course_experience/course-updates-fragment.html

```diff
@@ -36,7 +36,22 @@
         </div>
     </header>
     <div class="page-content">
-        ${HTML(updates_html)}
+        % if len(update_list_html) > 0:
```

Comment:
Nit: empty lists are falsey in Python, so you can do this (I prefer it, but not everyone does):

    % if update_list_html: